pose_evaluation/utils/conftest.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+import copy
+import itertools
+from pathlib import Path
+from typing import List, Dict, Tuple
+from pose_format import Pose
+import pytest
+from pose_evaluation.utils.pose_utils import load_pose_file
+from pose_format.utils.generic import fake_pose
+from pose_format.utils.openpose_135 import OpenPose_Components as openpose_135_components
+
+
+utils_test_data_dir = Path(os.path.dirname(os.path.realpath(__file__))) / 'test'/'test_data'
+
+@pytest.fixture(scope="function")
+def test_mediapipe_poses_paths()->List[Path]:
```

Comment:
add proper spacing ` -> `
if you use PyCharm, `Cmd+Shift+L` will fix all these issues